build.gradle.kts

```diff
@@ -26,7 +26,6 @@ dependencies {
     modImplementation("net.silkmc:silk-commands:$silkVersion")
 
     include(implementation("me.obsilabor", "alert", "1.0.8"))
-    include(implementation("org.jetbrains.kotlinx", "kotlinx-serialization-json", "1.7.0"))
```

Comment:
why?

Reply:
also, consider using one format of dependency notations and not multiple across the buildscript